src/WebJobs.Script/Workers/SharedMemoryDataTransfer/SharedMemoryMetadata.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
+{
+    public class SharedMemoryMetadata
+    {
+        public string Name { get; set; }
```

Comment:
Is offset needed here? I noticed when creating RpcSharedMemory - offset is set to `0` . intentional?

Reply:
This is a little for future planning (i.e. when adding the cache - we have a few ideas/cases where instead of reading the entire map we can read from an offset.)